haskell-open-source.md

```diff
@@ -64,3 +64,27 @@ To determine what changes are "breaking", use the following:
 ![](./pvp-chart.png)
 
 With `A=EPOCH`, `B=MAJOR`, and `C=MINOR`.
+
+## Release
+
+All (interested) Frecklers should be in the Maintainers group of our Hackage
+libraries, with permission to upload new versions.
+
+To release a new version:
+
+1. Update the package version
+1. Add a `CHANGELOG` entry
+1. Open the above in a Pull Request and get review
+1. Once merged, run the following from `main`:
+
+   ```console
+   stack upload --pvp-bounds both .
```

Comment:
I'm not a huge fan of `--pvp-bounds`, but I also don't have a better answer and don't want people to waste time on bounds, so... :shrug:

Reply:
Yeah, big ditto there.